src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/ImagePages/ImageUriVariationsPage.xaml

```diff
@@ -0,0 +1,754 @@
+<Page x:Class="Uno.UI.RuntimeTests.Images.ImageUriVariationsPage"
```

Comment:
It's generally best to have a single file per test, as loading this one will generate a lot of loading requests, making debugging difficult.

Reply:
We have a data driven test that covers most of these scenarios. This page just includes the same set but defined in XAML and provides quick way to validate all the combinations - more of an indicator loading/failing isn't working correctly on particular platforms